docker/dockerfiles/fe/fe_entrypoint.sh

```diff
@@ -88,22 +88,28 @@ probe_leader_for_pod0()
     local memlist=
     while true
     do
-        memlist=`show_frontends $svc`
-        local leader=`echo "$memlist" | grep '\<LEADER\>' | awk '{print $2}'`
-        if [[ "x$leader" != "x" ]] ; then
-            # has leader, done
-            log_stderr "Find leader: $leader!"
-            FE_LEADER=$leader
-            return 0
-        fi
+        if nc -z -4 -w 2 $svc $QUERY_PORT ; then
```

Comment:
```
NC="nc -z -w 2"
if $NC $svc $QUERY_PORT
```

`NC` can be alias for all the places that can be referenced when using `nc`

 , may need to remove `-4` since the ipv6 support is in progress.